### PR TITLE
Working additional service updating (stop, start etc)

### PR DESCRIPTION
**Pull request by ryan-shaw**
Rushed code at the moment... welcome suggestions on how to make it cleaner.

### CODE REVIEW

File: haproxy/haproxycfg.py [hunk 1]
```diff
@@ -86,7 +87,7 @@ def _init_new_links():
             if ADDITIONAL_SERVICES:
                 additional_services = ADDITIONAL_SERVICES.split(",")
                 NewLinkHelper.get_additional_links(docker, additional_services, haproxy_container,
-                                                   links, Haproxy.cls_linked_services)
+                                                   links, Haproxy.cls_linked_services, cls_additional_services)
```
Comment (tifayuki) [inline haproxy/haproxycfg.py]: Here will raise an error:INFO:haproxy:Error loading ADDITIONAL_SERVICES: global name 'cls_additional_services' is not defined

**Comment by tifayuki** [comment]
@ryan-shaw 
Thank you. This PR is useful. 

I think we can simple add additional services to `Haproxy.cls_linked_services`. Both directly linked services and additional services are the same to the haproxy, in my opinion.

**Comment by ryan-shaw** [comment]
@tifayuki Yes, you're right. I'll try and update this soon.

**Comment by tifayuki** [comment]
Fixed in https://github.com/docker/dockercloud-haproxy/commit/a37529e5764aa79ce350cbf0f6b6dc5810edf2e9

**Pull request closed without merging**